create_graphics.py

```diff
@@ -85,6 +85,7 @@ def create_zip(png_files, zipf):
             fd = open(lock_file, 'w')
             print(f'Writing to zip file for files like: {png_files[0][-10:]}')
 
+            print(f'trying zip file {zipf}')
```

Comment:
Would you mind combining this line with the one above to maybe just say "Writing to {zipf} for files like:..."?

Reply:
Done.  Thanks for spotting that.